pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -183,3 +186,36 @@ def test_dataframe_constructor_with_different_dtype_raises():
     xpr = "Cannot coerce extension array to dtype 'int64'. "
     with tm.assert_raises_regex(ValueError, xpr):
         pd.DataFrame({"A": arr}, dtype='int64')
+
+
+_ts = pd.Series(DecimalArray(make_data()))
+
+
+class TestOperator(BaseDecimal, TestSeriesOperators):
+    @cache_readonly
+    def ts(self):
+        ts = _ts.copy()
+        ts.name = 'ts'
+        return ts
+
+    def test_operators(self):
+        def absfunc(v):
+            if isinstance(v, pd.Series):
+                vals = v.values
+                return pd.Series(vals._from_sequence([abs(i) for i in vals]))
+            else:
+                return abs(v)
+        context = decimal.getcontext()
```

Comment:
what the heck is all this?

Reply:
I am reusing the tests that are in the class  `pandas/tests/series/test_operators/TestSeriesOperators`.  For decimal, the default when dividing by zero is to raise an error, as opposed to in numpy, where `inf` is returned.  So by changing the defaults for `Decimal`, we get the same behavior as numpy.

There is a related issue with respect to `abs()`, because the tests in `TestSeriesOperators` call `np.abs()`, but that doesn't work for the `Decimal` class.